build/quicklink.php

```diff
@@ -132,3 +170,37 @@ function quicklink_plugin_compatibility_files() {
 	}
 }
 add_action( 'init', 'quicklink_plugin_compatibility_files' );
+
+/**
+ * Add quicklink to the default scripts to make it available earlier in the runtime.
+ *
+ * @param WP_Scripts $scripts The WP_Scripts instance.
+ *
+ * @return void
+ */
+function quicklink_to_default_scripts( $scripts ) {
+	$scripts->add( 'quicklink', QUICKLINK_URL . 'quicklink.min.js', array(), '<##= pkg.version ##>', true );
```

Comment:
Passing `true` doesn't seem to work to force into the footer per https://github.com/luehrsenheinrich/wp-quicklink/pull/36/files#r566338633:

```suggestion
	$scripts->add( 'quicklink', QUICKLINK_URL . 'quicklink.min.js', array(), '<##= pkg.version ##>' );
```